app/controllers/api/v1/subnets_controller.rb

```diff
@@ -25,6 +25,7 @@ def show
       api :POST, '/subnets', 'Create a subnet'
       param :subnet, Hash, :required => true do
         param :name, String, :desc => 'Subnet name', :required => true
+        param :type, String, :desc => 'Type or protocol, Subnet::Ipv4 or Subnet::Ipv6, defaults to Subnet::Ipv4'
```

Comment:
You _might_ find it easier to implement if calling it something other than `type` (addressing? addressing_type? network_type?)